### PR TITLE
k9copy: init at 3.0.3

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

**Comment by geerds** [comment]
Could you please also define `meta.platforms` ? (e.g. `platforms = platforms.linux`)

**Comment by flosse** [comment]
> Could you please also define `meta.platforms`?

sure

File: pkgs/applications/video/k9copy/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    kdesu
+    kdoctools
+  ]);
+
+  meta = {
+    description = "DVD backup and DVD authoring program";
+    homepage = "http://k9copy-reloaded.sourceforge.net/";
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = with stdenv.lib.maintainers; [ flosse ];
+    platforms = stdenv.lib.platforms.linux;
```
Comment (peti) [inline pkgs/applications/video/k9copy/default.nix]: This program cannot be compiled on platforms other than Linux, i.e. Darwin?

Comment (flosse) [inline pkgs/applications/video/k9copy/default.nix]: hmm... i have no idea :-\ wikipedia tells me "Unix-like" but I have no chance to test it on OS X.

**Comment by geerds** [comment]
Sometimes I have the same problem. I don't know if a application also works on other platforms (like Darwin). Maybe we should introduce a new tag? Something like `needs-testing` in combination with `darwin` (or a different platform)

File: pkgs/applications/video/k9copy/default.nix (hunk 1 above)
Comment (peti) [inline pkgs/applications/video/k9copy/default.nix]: If the exact set of supported platforms is unknown, then it's probably better to specify meta.hydraPlatforms instead.

**Comment by jagajaga** [comment]
Please rename commit as it said in `CONTIBUTING.md`.

**Comment by lethalman** [comment]
Did you try this before merging? It doesn't even evaluate:

```
error: undefined variable ‘scope’ at "/home/user/nixpkgs/pkgs/applications/video/k9copy/default.nix":38:5
```

**Comment by flosse** [comment]
yes, there seems to be a change since 3113d31070f0a0d4a1cdbad2336e32c2a2312031 but I didn't find the right place :-\
@ttuegel might help us?

However, I fixit it  with these changes: https://github.com/flosse/nixpkgs/commit/86dbc9b5e2f11de545a64cd17afd5d76e34081ac (I'll create a PR as soon as the compiling on my local machine is done)

**Comment by flosse** [comment]
seems to work: https://github.com/NixOS/nixpkgs/pull/8517

**Comment by ttuegel** [comment]
Yes, the KDE Frameworks expressions no longer export implementation details; it is safe to just to add whatever packages you need to the `buildInputs`.

**Pull request merged**